CMakeLists.txt

```diff
@@ -125,82 +155,71 @@ target_link_libraries(nemsio::nemsio INTERFACE w3emc::w3emc_d bacio::bacio_4)
 find_package(Python 3 REQUIRED COMPONENTS Interpreter)
 message("Found Python: ${Python_EXECUTABLE}")
 
-###############################################################################
-### Checks
-###############################################################################
-if(DATM AND NOT S2S)
-   message(FATAL_ERROR "DATM=ON and S2S=OFF are incompatible, ABORT!")
-endif()
-
-if(S2S AND 32BIT)
-   message(FATAL_ERROR "S2S=ON and 32BIT=ON are incompatible, ABORT!")
-endif()
-
 ###############################################################################
 ### FMS
 ###############################################################################
-include(fms_files.cmake)
-add_library(fms ${fms_src_files} ${fms_headers})
-
-list(APPEND _fms_defs_public use_libMPI
-                             use_netCDF
-                             GFS_PHYS
-                             INTERNAL_FILE_NML)
-if(QUAD_PRECISION)
-  list(APPEND _fms_defs_public ENABLE_QUAD_PRECISION)
-endif()
+if(FMS)
+  include(fms_files.cmake)
```

Comment:
This block will be removed in ufs-community#372

Reply:
It is simplified in #372 but not removed.